client/ui/qml/Pages2/PageSetupWizardApiServicesList.qml

```diff
@@ -16,83 +16,88 @@ PageType {
 
     defaultActiveFocusItem: focusItem
 
-    FlickableType {
-        id: fl
+    ColumnLayout {
+        id: header
+
         anchors.top: parent.top
-        anchors.bottom: parent.bottom
-        contentHeight: content.height
+        anchors.left: parent.left
+        anchors.right: parent.right
 
-        ColumnLayout {
-            id: content
+        spacing: 0
 
-            anchors.top: parent.top
-            anchors.left: parent.left
-            anchors.right: parent.right
+        Item {
+            id: focusItem
+            KeyNavigation.tab: backButton
+        }
 
-            spacing: 0
+        BackButtonType {
+            id: backButton
+            Layout.topMargin: 20
+//                KeyNavigation.tab: fileButton.rightButton
+        }
 
-            Item {
-                id: focusItem
-                KeyNavigation.tab: backButton
-            }
+        HeaderType {
+            Layout.fillWidth: true
+            Layout.topMargin: 8
+            Layout.rightMargin: 16
+            Layout.leftMargin: 16
+            Layout.bottomMargin: 16
 
-            BackButtonType {
-                id: backButton
-                Layout.topMargin: 20
-//                KeyNavigation.tab: fileButton.rightButton
-            }
+            headerText: qsTr("VPN by Amnezia")
+            descriptionText: qsTr("Choose a VPN service that suits your needs.")
+        }
+    }
 
-            HeaderType {
-                Layout.fillWidth: true
-                Layout.topMargin: 8
-                Layout.rightMargin: 16
-                Layout.leftMargin: 16
-                Layout.bottomMargin: 32
+    ListView {
+        id: servicesListView
+        anchors.top: header.bottom
+        anchors.right: parent.right
+        anchors.left: parent.left
+        anchors.bottom: parent.bottom
+        anchors.topMargin: 16
+        anchors.bottomMargin: 16
+        spacing: 16
 
-                headerText: qsTr("VPN by Amnezia")
-                descriptionText: qsTr("Choose a VPN service that suits your needs.")
-            }
+        currentIndex: 1
+        clip: true
+        model: ApiServicesModel
+
```

Comment:
```suggestion
reuseItems: true
```

Reply:
I don't think it needs to be enabled for this listview